New update breaks (again) Monica script

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

Monica

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/monica.sh)"

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 📈 Which Proxmox version are you on?

pve-manager/8.4.14/b502d23c55afcba1 (running kernel: 6.8.12-15-pve)

### 📝 Provide a clear and concise description of the issue.

Can't install Monica anymore by executing the script on the VE shell

### 🔄 Steps to reproduce the issue.

1. Run the Monica script in the VE shell
2. The script breaks inside the "Configuring monica" phase

### ❌ Paste the full error output (if available).

```shell
  ✔️   Setup MariaDB 12.1.1
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Package 'nodejs' is not installed, so not removed
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.

removed 9 packages, and changed 42 packages in 3s

25 packages are looking for funding
  run `npm fund` for details
  ✔️   Setup Node.js 22

added 1 package in 718ms
  ✔️   Installed Node.js modules: yarn@latest
  ✔️   Set up MariaDB
  ✔️   Deployed: monica (4.1.2)
  ⏳   Configuring monica> App\Helpers\ComposerScripts::preInstall
Installing dependencies from lock file
Verifying lock file contents can be installed on current platform.
Nothing to install, update or remove
Package php-http/message-factory is abandoned, you should avoid using it. Use psr/http-factory instead.
Package web-token/jwt-key-mgmt is abandoned, you should avoid using it. Use web-token/jwt-library instead.
Package web-token/jwt-signature-algorithm-ecdsa is abandoned, you should avoid using it. Use web-token/jwt-library instead.
Package web-token/jwt-signature-algorithm-eddsa is abandoned, you should avoid using it. Use web-token/jwt-library instead.
Package web-token/jwt-signature-algorithm-rsa is abandoned, you should avoid using it. Use web-token/jwt-library instead.
Package nunomaduro/larastan is abandoned, you should avoid using it. Use larastan/larastan instead.
Generating optimized autoload files
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover

   INFO  Discovering packages.  

  asbiin/laravel-adorable ............................................... DONE
  asbiin/laravel-webauthn ............................................... DONE
  creativeorange/gravatar ............................................... DONE
  intervention/image .................................................... DONE
  laravel/cashier ....................................................... DONE
  laravel/passport ...................................................... DONE
  laravel/socialite ..................................................... DONE
  laravel/ui ............................................................ DONE
  laravolt/avatar ....................................................... DONE
  mariuzzo/laravel-js-localization ...................................... DONE
  monicahq/laravel-cloudflare ........................................... DONE
  monicahq/laravel-sabre ................................................ DONE
  nesbot/carbon ......................................................... DONE
  nunomaduro/termwind ................................................... DONE
  pragmarx/google2fa-laravel ............................................ DONE
  rinvex/countries ...................................................... DONE
  sentry/sentry-laravel ................................................. DONE
  stevebauman/location .................................................. DONE
  vinkla/hashids ........................................................ DONE
  vluzrmos/language-detector ............................................ DONE
  werk365/etagconditionals .............................................. DONE

97 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
> Illuminate\Foundation\ComposerScripts::postInstall
yarn install v1.22.22
[1/5] Validating package.json...
error @: The engine "node" is incompatible with this module. Expected version "20.x". Got "22.19.0"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

[ERROR] in line 47: exit code 0: while executing command $STD yarn install

[ERROR] in line 1345: exit code 0: while executing command lxc-attach -n "$CTID" -- bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/install/${var_install}.sh)"
```

### 🖼️ Additional context (optional).

_No response_

--- a/ct/monica.sh
+++ b/ct/monica.sh
@@ -46,6 +46,7 @@ function update_script() {
     cp -r /opt/monica-backup/.env /opt/monica
     cp -r /opt/monica-backup/storage/* /opt/monica/storage/
     $STD composer install --no-interaction --no-dev
+    $STD yarn config set ignore-engines true
     $STD yarn install
     $STD yarn run production
     $STD php artisan monica:update --force

--- a/install/monica-install.sh
+++ b/install/monica-install.sh
@@ -44,6 +44,7 @@ sed -i -e "s|^DB_USERNAME=.*|DB_USERNAME=${DB_USER}|" \
   -e "s|^HASH_SALT=.*|HASH_SALT=${HASH_SALT}|" \
   /opt/monica/.env
 $STD composer install --no-dev -o --no-interaction
+$STD yarn config set ignore-engines true
 $STD yarn install
 $STD yarn run production
 $STD php artisan key:generate

